app/models/inbox_result.rb

```diff
@@ -5,6 +5,8 @@ class InboxResult < ApplicationRecord
 
   # see result.rb for explanation of the scope
   belongs_to :inbox_person, ->(ibr) { where(competition_id: ibr.competition_id) }, primary_key: :id, foreign_key: :person_id, optional: true
+  delegate :country_iso2, to: :person
+  delegate :wca_id, to: :person
```

Comment:
These two methods are now becoming a part of `DEFAULT_SERIALIZE_OPTIONS`. Which means they will be called on every single InboxResult.

The comment on `def person` implies that loading the person is very cost-prohibitive, so it looks to me like doing this now will create an incredible amount of SQL queries :(

Reply:
This is part of the reason why the old Ruby templating code uses `persons_by_id` as a lookup, instead of actually accessing each person on the individual result row during iteration.
You will have to come up with a similar solution to reduce the database load.

Reply:
Oh okay, I did few optimizations now. Hope it's better.